physical/raft/raft.go

```diff
@@ -666,6 +681,7 @@ func (b *RaftBackend) SetupCluster(ctx context.Context, opts SetupOpts) error {
 
 	// Setup the raft config
 	raftConfig := raft.DefaultConfig()
+	raftConfig.SnapshotInterval = 5 * time.Second
```

Comment:
What's the rational for setting this to 5s?

Reply:
Can't set it too high or we get a timeout on the writer side.  I wanted it higher than the stabilization time so that we'd see that node continue to not be promoted for a bit after the stabilization+reconciliation had occurred, then eventually see it promoted.